src/Infrastructure.EntityFramework/Repositories/CollectionRepository.cs

```diff
@@ -427,7 +484,64 @@ public async Task<ICollection<CollectionDetails>> GetManyByUserIdAsync(Guid user
         {
             var dbContext = GetDatabaseContext(scope);
             var query = CollectionAdminDetailsQuery.ByCollectionId(collectionId, userId).Run(dbContext);
-            var collectionDetails = await query.FirstOrDefaultAsync();
+
+            CollectionAdminDetails collectionDetails;
+
+            // SQLite does not support the GROUP BY clause
+            if (dbContext.Database.IsSqlite())
+            {
+                collectionDetails = (await query.ToListAsync())
+                    .GroupBy(c => new
+                    {
+                        c.Id,
+                        c.OrganizationId,
+                        c.Name,
+                        c.CreationDate,
+                        c.RevisionDate,
+                        c.ExternalId
+                    }).Select(collectionGroup => new CollectionAdminDetails
+                    {
+                        Id = collectionGroup.Key.Id,
+                        OrganizationId = collectionGroup.Key.OrganizationId,
+                        Name = collectionGroup.Key.Name,
+                        CreationDate = collectionGroup.Key.CreationDate,
+                        RevisionDate = collectionGroup.Key.RevisionDate,
+                        ExternalId = collectionGroup.Key.ExternalId,
+                        ReadOnly = Convert.ToBoolean(collectionGroup.Min(c => Convert.ToInt32(c.ReadOnly))),
```

Comment:
The conversions are unnecessary:

* `collectionGroup.All(c => c.ReadOnly)` (equivalent of MIN, it will return false if there is any `ReadOnly = false`)
* `collectionGroup.Any(c => c.Manage)` (equivalent of MAX, it will return true if there is any `Manage = true`)

However I can see this is how we do it in existing queries, and that does more closely track the SQL equivalent, so I'll leave it up to you as to which you prefer.

Reply:
That's a fair point. I had gone with what already existed in the other queries, as I figured there was some underlying implementation detail forcing the extra conversions (e.g. maybe one provider doesn't behave properly with `Any` or `All`).

I think I'll opt to leave it alone for now since we know it works as expected and it keeps consistent with the other queries.